src/lib/redux-store/utils/selector-utils.ts

```diff
@@ -1,46 +1,22 @@
 import type { RillReduxState } from "$lib/redux-store/store-root";
-import type { EntityRecord } from "$common/data-modeler-state-service/entity-state-service/EntityStateService";
 
-export function generateBasicSelectors(sliceKey: keyof RillReduxState) {
+function generateCommonSelectors<Entity>(sliceKey: keyof RillReduxState) {
```

Comment:
Could you add a JSDoc comment here explaining what this does? It makes reading the code much easier within the editor